### PR TITLE
Add links to the cm api integration guide

### PR DESCRIPTION
**Pull request by agektmr**

### CODE REVIEW

File: src/content/en/updates/2016/04/credential-management-api.md [hunk 1]
```diff
@@ -365,9 +365,12 @@ You may override `navigator.credentials` and hook it to your Chrome Extension to
 `get()` or  `store()` credentials.
 
 ## Resources
+To learn more about the Credential Management API, checkout [Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/).
```
Comment (petele) [inline src/content/en/updates/2016/04/credential-management-api.md]: For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`

File: src/content/en/updates/2016/04/credential-management-api.md [hunk 2]
```diff
@@ -365,9 +365,12 @@ You may override `navigator.credentials` and hook it to your Chrome Extension to
...
+
 * [API Specification](https://www.w3.org/TR/credential-management/)
 * [Spec discussions & feedback](https://github.com/w3c/webappsec-credential-management)
 * [MDN API Reference](https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API)
+* [Credential Management API Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/)
```
Comment (petele) [inline src/content/en/updates/2016/04/credential-management-api.md]: For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`

File: src/content/ja/updates/2016/04/credential-management-api.md [hunk 3]
```diff
@@ -353,9 +353,12 @@ mediation モードを有効にすることができます。
 それをあなたの Chrome 拡張機能にフックすることはできるかもしれません。
 
 ## リソース
+より深く Credential Management API を学ぶには、[Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/) をご覧下さい。
```
Comment (petele) [inline src/content/ja/updates/2016/04/credential-management-api.md]: For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`

File: src/content/ja/updates/2016/04/credential-management-api.md [hunk 4]
```diff
@@ -353,9 +353,12 @@ mediation モードを有効にすることができます。
...
+
 * [API 仕様](https://www.w3.org/TR/credential-management/)
 * [仕様のディスカッションとフィードバック](https://github.com/w3c/webappsec-credential-management)
 * [MDN API リファレンス](https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API)
+* [Credential Management API インテグレーションガイド](https://developers.google.com/web/fundamentals/security/credential-management/)
```
Comment (petele) [inline src/content/ja/updates/2016/04/credential-management-api.md]: For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`

**Comment by agektmr** [comment]
Nailed.

**Comment by petele** [comment]
Merged - will push live Monday.

**Pull request merged**